typescript/submissionEvents.d.ts

```diff
@@ -440,6 +440,11 @@ export type SharepointStoreFilesSubmissionEvent = FormEventBase & {
        * forward slash and not end with a forward slash e.g. "/forms/1/submissions"
        */
       folderPath?: string
+      /**
+       * Set to `true` to prevent all form submission attachments from being
+       * uploaded to SharePoint.
+       */
+      excludeAttachments: boolean
```

Comment:
Would it be better to make this the default and reverse this? `includeAttachments?: boolean`

Reply:
Blake wants users to opt out of uploading attachments

Reply:
I had in mind more of excluding elementIds like the PDF configuration

Reply:
Would be nice to be more granular. Lets discuss with Blake.